tests/build/jvm-build.go

```diff
@@ -99,16 +98,6 @@ var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", func() {
 		if err != nil {
 			klog.Infof("error deleting task git-clone", err.Error())
 		}
-		trList, err := f.TektonController.ListAllTaskRuns(appStudioE2EApplicationsNamespace)
```

Comment:
deleting this is overkill in my opinion since this is just cleanup code

Reply:
that said, I think you should comment this out vs. delete .... we should not depend on the controller's pruning catching all the task runs, especially when we are pruning artifactbuilds and dependencybuilds right before